src/mqt/bench/evaluation/evaluation.py

```diff
@@ -57,3 +56,12 @@ def evaluate_qasm_file(filename: str) -> EvaluationResult:
         "parallelism": parallelism,
         "liveness": liveness,
     }
+    return cast(EvaluationResult, res_dict)
```

Comment:
No need to cast here. Just do
```suggestion
    return EvaluationResult(
        filename=filename,
        num_qubits=int(str(filename).split("_")[-1].split(".")[0]),
        depth=qc.depth(),
        etc.
    )
```
And at that point, you can use a data class as well, which would look exactly the same here.

Reply:
Adjusted the code accordingly.